predicates/interval/interval.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2015 Zalando SE
```

Comment:
Looks like that since last time I was looking into our guidelines something has changed. Now it is not mandatory. I'll remove it.